utils.js

```diff
@@ -323,7 +323,9 @@ export const getNpmMetadata = async function (pkgList) {
         metadata_cache[key] = body;
       }
       p.description = body.description;
-      p.license = body.license;
+      if (!p.license) {
```

Comment:
How about we check for empty strings as well?

```
if (!p.license || p.license.trim() === "") {
```

Reply:
I've reverted this change, so this comment no longer applies.